src/check/auditChecker.js

```diff
@@ -20,6 +20,12 @@ module.exports = {
                     if (resolution === RESOLUTIONS.NONE) {
                         unresolved = true
                     }
+                    if (resolution === RESOLUTIONS.REASON_MISSING) {
```

Comment:
I'm not convinced this should be exposed as a resolution. I'd rather add guards against proceeding in the interactive wizard part and fail resolve file validation as a critical error if valid reason is missing form the file - with a good descriptive error.

Reply:
👍, following up in https://github.com/naugtur/audit-resolve-core/pull/5#discussion_r911455086